### PR TITLE
Phone number refactor

### PR DESCRIPTION
**Pull request by afjlambert**
This PR fixes the styling of the Phone number on our app. The number is now always a link with tel:. The markdown is also parsed and rendered by the LinkRenderer which detects tel: in the href. For every link we now also track events. This is even the case for non-telephone links in markdown. Which is just a bonus of this PR. The styling for non-telephone links is also fixed because it uses the same Anchor.

I had to add an Atom because the Link component didn't work for mee, see source-comments.
Another optimization is the deduplication of text, now available in `ContactSentence`.
We don't want to bother content editors with linking the phone number everywhere, so the conversion of phone-text -> phone-link is now done by the sttr-build script. Had to update the sttr files accordingly.
Used Typescript where it was easy to add/convert.

IntroPage (hardcoded) before:
Schermafbeelding 2020-08-21 om 10 37 45

IntroPage (hardcoded) after (print screen shows "vakantiehuis?Bel", that's now fixed with an extra space):
Schermafbeelding 2020-08-21 om 10 37 52

Markdown from STTR with normal link, before:
Schermafbeelding 2020-08-21 om 10 38 10

Markdown from STTR with normal link, after:
Schermafbeelding 2020-08-21 om 10 38 15

Conclusion from markdown:
Schermafbeelding 2020-08-21 om 10 38 59

Footer still works as expected:
Schermafbeelding 2020-08-21 om 10 39 07

Some matomo measurement examples:
Schermafbeelding 2020-08-21 om 10 38 49
Schermafbeelding 2020-08-21 om 10 39 22
Schermafbeelding 2020-08-21 om 10 39 33

**Please squash and merge.**

### CODE REVIEW

File: apps/client/src/components/Conclusion.js [hunk 1]
```diff
@@ -108,7 +109,15 @@ const Conclusion = ({ checker, topic: { slug } }) => {
...
         <Alert
           heading="Let op"
-          content={`De vergunningcheck is nog in ontwikkeling. Hierdoor kunnen wij nog geen zekerheid bieden dat de uitkomst correct is. Ook is de informatie nog niet voor iedereen goed te lezen of te beluisteren. Wilt u iets zeker weten of wilt u meer informatie? Bel het telefoonnummer 14 020, maandag tot en met vrijdag van 08.00 uur tot 18.00 uur.`}
+          content={
+            <>
+              De vergunningcheck is nog in ontwikkeling. Hierdoor kunnen wij nog
+              geen zekerheid bieden dat de uitkomst correct is. Ook is de
+              informatie nog niet voor iedereen goed te lezen of te beluisteren.
+              Wilt u iets zeker weten of wilt u meer informatie?{" "}
+              <ContactSentence link={false} />
```
Comment (robinpiets) [inline apps/client/src/components/Conclusion.js:118]: Nice!

File: apps/client/src/components/Markdown/renderers/LinkRenderer.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { useMatomo } from "@datapunt/matomo-tracker-react";
```
Comment (robinpiets) [inline apps/client/src/components/Markdown/renderers/LinkRenderer.tsx]: I like this file!

File: apps/client/src/config/matomo.js [hunk 3]
```diff
@@ -13,10 +13,13 @@ export const matomo = {
   siteId: getMatomoSiteId(isProduction),
 };
 
+export const trackingEnabled = () => !localStorage.getItem("doNotTrack");
```
Comment (robinpiets) [inline apps/client/src/config/matomo.js:16]: 👍

File: apps/client/src/intros/DakkapelIntro.js [hunk 4]
```diff
@@ -12,14 +13,12 @@ export default () => (
     <List variant="bullet">
       <ListItem>
         Wilt u de dakkapel plaatsen op een woonwagen, een tijdelijk gebouw, een
-        blokhut of een vakantiehuis? Bel dan de gemeente op{" "}
-        <a href="tel:14020">14 020</a>, maandag tot en met vrijdag van 08.00 uur
-        tot 18.00 uur
+        blokhut of een vakantiehuis?{" "}
+        <ContactSentence eventName="Intro - Extraordinary situation" />
```
Comment (robinpiets) [inline apps/client/src/intros/DakkapelIntro.js]: This we will change in a separate PR @svenjens

File: apps/client/sttr_build/index.js [hunk 5]
```diff
@@ -55,6 +55,15 @@ function checkStatus(res) {
   return res;
 }
 
+/**
+ * Preprocessors are used to change the incoming xml. The xml
+ * is not yet parsed, so they are basic find replace functions.
+ * The output of every preprocessor is passed as input to the next.
+ */
+const preprocessors = [
+  (str) => str.replace(/ 14\s?020/g, " [14 020](tel:14020)"), // Fix phone-number
```
Comment (robinpiets) [inline apps/client/sttr_build/index.js]: This comment can be updated to something like "Replace phone number with Link"

**Review by robinpiets — APPROVED** [review]

File: apps/client/src/components/Markdown/renderers/LinkRenderer.tsx (hunk 2 above)
Comment (afjlambert) [reply apps/client/src/components/Markdown/renderers/LinkRenderer.tsx]: 😃

File: apps/client/src/intros/DakkapelIntro.js (hunk 4 above)
Comment (afjlambert) [reply apps/client/src/intros/DakkapelIntro.js]: 👍

**Comment by robinpiets** [comment]
New screenshots @afjlambert :

image

image

image

image

**Review by svenjens — APPROVED** [review]

File: apps/client/sttr_build/index.js (hunk 5 above)
Comment (afjlambert) [reply apps/client/sttr_build/index.js]: Check

**Review by svenjens — APPROVED** [review]

**Review by robinpiets — APPROVED** [review]

**Pull request merged**